datasets/peer_read/peer_read.py

```diff
@@ -0,0 +1,306 @@
+                    "accepted": datasets.Value("bool"),
+                    "abstract": datasets.Value("string"),
+                    "histories": datasets.features.Sequence(datasets.features.Sequence(datasets.Value("string"))),
+                    "reviews": datasets.features.Sequence(
+                        {
+                            "date": datasets.Value("string"),
+                            "title": datasets.Value("string"),
+                            "other_keys": datasets.Value("string"),
+                            "originality": datasets.Value("string"),
+                            "comments": datasets.Value("string"),
+                            "is_meta_review": datasets.Value("bool"),
+                            "is_annotated": datasets.Value("bool"),
+                            "recommendation": datasets.Value("string"),
+                            "replicability": datasets.Value("string"),
+                            "presentation_format": datasets.Value("string"),
+                            "clarity": datasets.Value("string"),
+                            "meaningful_comparison": datasets.Value("string"),
+                            "substance": datasets.Value("string"),
+                            "reviewer_confidence": datasets.Value("string"),
+                            "soundness_correctness": datasets.Value("string"),
+                            "appropriateness": datasets.Value("string"),
+                            "impact": datasets.Value("string"),
+                        }
+                    ),
+                }
+            )
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            supervised_keys=None,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+        url = _URLs["dataset_repo"]
+        data_dir = dl_manager.download_and_extract(url)
+        paths = self._get_paths(
+            data_dir=data_dir,
+            domain=self.config.name,
+        )
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepaths": paths["train"],
+                    "split": "train",
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.TEST,
+                gen_kwargs={"filepaths": paths["test"], "split": "test"},
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.VALIDATION,
+                gen_kwargs={
+                    "filepaths": paths["dev"],
+                    "split": "dev",
+                },
+            ),
+        ]
+
+    def _generate_examples(self, filepaths, split):
+        """ Yields examples. """
+        for id_, filepath in enumerate(filepaths):
```

Comment:
```suggestion
        for id_, filepath in enumerate(sorted(filepaths)):
```
we need to make the order deterministic here

Reply:
Sure